src/Ocelot/Configuration/Validator/FileConfigurationFluentValidator.cs

```diff
@@ -115,6 +120,26 @@ private static bool IsPlaceholderNotDuplicatedIn(string pathTemplate)
             return placeholders.Count == placeholders.Distinct().Count();
         }
 
+        private static bool ServiceFabricPlaceholdersInServiceName(FileConfiguration configuration, FileRoute route)
```

Comment:
@raman-m Why did you squeeze that in the Hotfix?

Reply:
It validates Service Fabric cases only.
The regular cases have no validation rules at all.
Is it out of the scope of hotfix?

Reply:
>why did you push a service fabric validation mechanism too?

Because I published this feature in docs when worked on 928 vs 1927.
 🆗 I will remove the code from this PR